lstchain/reco/tests/test_utils.py

```diff
@@ -2,6 +2,7 @@
 import astropy.units as u
 from astropy.time import Time
 from astropy.coordinates import SkyCoord
+from ctapipe.containers import EventType
```

Comment:
Where is this being used?

Reply:
Thanks! It was used at some point, but no longer.